dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceFormatter.java

```diff
@@ -736,13 +1070,100 @@ private String getMarker(LogRecord logRecord) {
     }
 
     /**
+     * check casting to WsLogRecord
+     *
      * @return
      */
-    private WsLogRecord getWsLogRecord(LogRecord logRecord) {
+    //change to public
+    public WsLogRecord getWsLogRecord(LogRecord logRecord) {
         try {
             return (WsLogRecord) logRecord;
         } catch (ClassCastException ex) {
             return null;
         }
     }
+
+    /**
+     * Outputs filteredStream of genData
+     *
+     * @param genData object to filter
+     * @return filtered message of the genData
+     */
+    protected String filteredStreamOutput(GenericData genData) {
+        String txt = null;
+        String loglevel = null;
+        KeyValuePair kvp = null;
+
+        ArrayList<Pair> pairs = genData.getPairs();
+        for (Pair p : pairs) {
+
+            if (p instanceof KeyValuePair) {
+
+                kvp = (KeyValuePair) p;
+                if (kvp.getKey().equals("message")) {
+                    txt = kvp.getValue();
+                } else if (kvp.getKey().equals("loglevel")) {
+                    loglevel = kvp.getValue();
+                } else if (kvp.getKey().equals("throwable")) {
+                    loglevel = kvp.getValue();
```

Comment:
this line seems odd... just want to double check to make sure you meant to set loglevel to the value of throwable